### DISCUSSION TITLE
Pin assignment changed after upgrade to 9.4.0

### DISCUSSION THREAD

**Discussion by nonix** [arendst/Tasmota#12001] [Q&A] [answered]
Hi guys, not sure if this is a bug or by design. However, I was testing my rotary encoder assigning it GPIO0 and GPIO2 resp. (A & B). That was on FW. 9.3.1 after upgrade to 9.4.0 the GPIO2 has changed to **Rotary B_n**

I am just asking before opening an issue :-) 

Regs,
 N.

**Comment by ascillato2** [comment, marked as answer]
Hi, that shouldn't have happened. Did you used a custom version of Tasmota 9.3.1 or you have used 9.3.1 and 9.4 both 8n precompiled versions?

**Reply by nonix** [reply]
Hi Adrian, yes it could have been me. The 9.3.1 was my modified/compiled version.
I did try to reproduce the issue with std. stock version of 9.3.1. followed by upgrade to 9.4 and could not reproduce. the pin assignment was as it should be.
My apology for time wasting. 
N.
